README.md

```diff
@@ -1,3 +1,11 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+Welcome to my NC News API Project!
+
+## Getting Started
+
+To run this in your local machine, create two .env files; one for test and one for development.
```

Comment:
Nice start!
What naming conventions should the .env files have? This would be beneficial to add.
Where can the database names be found?